src/hooks/useIsSnapInstalled/useIsSnapInstalled.tsx

```diff
@@ -35,6 +35,16 @@ export const checkIsMetaMask = pMemoize(
   },
 )
 
+export const checkisMetaMaskMobileWebView = () => {
```

Comment:
Now that we have this, we are able to disable the snaps modal showing up on MM mobile

Reply:
91439d2cc11e3be4905aa39b500990025c0f8754